x/examples/outline-pwa/README.md

```diff
@@ -1,19 +1,345 @@
-# Outline PWA Wrapper
+# Building a Censorship-Resistant Android/iOS App with CapacitorJS and the Outline SDK Mobileproxy
 
-Demonstration of how to wrap any URL with CapacitorJS in a way that all content is loaded over Outline SDK's Mobileproxy. Turn your website into a censorship-resistant app!
+This code lab guides you through creating a censorship-resistant Android/iOS app from scratch that wraps your Progressive Web App (PWA) using CapacitorJS and the Outline SDK Mobileproxy. This setup routes your site's traffic through the Outline proxy, bypassing network restrictions. Note that the source in this folder is representative of what your end product should look like once the code lab is completed.
 
-TODO: step-by-step walkthrough of the app-building process
+**Prerequisites:**
 
-## Setup
+* A website you want to make censorship resistant.
+* Make sure your development environment is set up with the following. **[Please follow CapacitorJS's environment setup guide](https://capacitorjs.com/docs/getting-started/environment-setup)**
+  * [Node.js](https://nodejs.org/en/), for the Capacitor build system.
+  * [GoLang](https://go.dev/), to build the Outline Mobile Proxy.
+  * For Android, [OpenJDK 17](https://stackoverflow.com/a/70649641) and [Android Studio](https://developer.android.com/studio/)
+  * For iOS, [XCode](https://developer.apple.com/xcode/) and [cocoapods](https://cocoapods.org/)
+* [Wireshark](https://www.wireshark.org/), to confirm the app is working
```

Comment:
"to confirm the app is working" doesn't seem quite correct. The app can work even if the mobile proxy doesn't, right? Maybe "to confirm traffic is going through the Outline proxy" or something?

Reply:
Also, Wireshark is only used in the Android instructions. Maybe mark it as "(Android only)".